packages/react-storage/src/components/StorageBrowser/Views/LocationDetailView/Controls.tsx

```diff
@@ -24,13 +26,37 @@ export const Title = (): React.JSX.Element => {
   return <TitleElement>{title}</TitleElement>;
 };
 
+const LocationDetailViewRefresh = () => {
+  const [{ history }] = useControl({
+    type: 'NAVIGATE',
+  });
+
+  const [{ data, isLoading }, handleList] = useAction({
+    type: 'LIST_LOCATION_ITEMS',
+  });
+
+  const prefix = history.join('');
+
+  return (
+    <Refresh
+      disabled={isLoading || data.result.length <= 0}
+      onClick={() =>
+        handleList({ prefix, options: { refresh: true, pageSize: 1000 } })
+      }
+    />
+  );
+};
+
 // @ts-expect-error TODO: add Controls assignment
 export const LocationDetailViewControls: LocationDetailViewControls = () => {
   return (
     <>
       <Navigate />
       <Title />
-      <ActionSelect />
+      <div className={`${CLASS_BASE}__action-controls`}>
```

Comment:
I think the `refresh` and `action-menu` both have the same CSS grid placement because on the locations view, refresh should be showing on the last column since their isn't an action-menu

Do you have any thoughts on how to conditionally change it? I don't mind removing it, just wanted to have this set up so it was easier to test without having to modify the CSS in dev tools so that the refresh button isn't underneath the action menu